src/styles/blocks/card.scss

```diff
@@ -0,0 +1,142 @@
+@import "../utils/variables.scss";
+
+.catalog {
+  padding-top: 50px;
+  display: grid;
+  grid-template-columns: repeat($column-count, $width-card);
+  grid-template-rows: repeat(2, 1fr);
+  gap: 46px 48px;
+  justify-content: center;
+
+  @media (max-width: 487px) {
+    $column-count: 1;
+    grid-template-columns: repeat($column-count, $width-card);
+  }
+
+  @media (min-width: 488px) {
+    $column-count: 2;
+    grid-template-columns: repeat($column-count, $width-card);
+  }
+
+  @media (min-width: 768px) {
+    $column-count: 3;
+    grid-template-columns: repeat($column-count, $width-card);
+  }
+
+  @media (min-width: 1024px) {
+    $column-count: 4;
+    grid-template-columns: repeat($column-count, $width-card);
+  }
+}
+
+.card {
+  width: $width-card;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  transition: 0.3s;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image-container {
+    margin: 0 3px 38px;
+  }
+
+  &__image {
+    width: 100%;
+    height: 98%;
+    object-fit: contain;
```

Comment:
Try to set image dimensions same as in figma. Maybe it will fix the image diff issue in the test report